lib/rucio/daemons/bb8/t2_background_rebalance.py

```diff
@@ -52,7 +52,7 @@
 rses_under_ratio = sorted([rse for rse in rses if rse['ratio'] < global_ratio - global_ratio * tolerance], key=lambda k: k['ratio'], reverse=False)
 
 session = get_session()
-active_rses = session.query(models.ReplicationRule.rse_expression).filter(or_(models.ReplicationRule.state == RuleState.REPLICATING, models.ReplicationRule.state == RuleState.STUCK),
```

Comment:
breaking this line in several will improve the comprehension and reading the code.